app/controllers/about_controller.rb

```diff
@@ -0,0 +1,2 @@
+class AboutController < PagesController
```

Comment:
We have an existing `PagesController`. We're using this to serve up any static pages, such as index and now /about. 

Instead of a new controller, create a `about` method in the PagesController.